tests/libnode/node.go

```diff
@@ -443,3 +445,14 @@ func GetNodeHostModel(node *k8sv1.Node) (hostModel string) {
 	}
 	return hostModel
 }
+
+func ExecuteCommandOnNodeThroughVirtHandler(
+	virtCli kubecli.KubevirtClient,
```

Comment:
There is no need to pass `kubecli.KubevirtClient` as you can use `kubevirt.Client()` directly, it's a singleton.

Reply:
Done